docs/copilot/getting-started-chat.md

```diff
@@ -15,140 +15,182 @@ If you're new to using GitHub Copilot in VS Code, see the [GitHub Copilot Gettin
+* To use GitHub Copilot, you must have an active subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
 
-## Install the GitHub Copilot Chat extension
+Follow these steps to [Set up GitHub Copilot in VS Code](/docs/copilot/getting-started.md#set-up-vs-code-for-github-copilot).
 
-The Chat features are available by installing the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extension.
+## Get your first Copilot chat conversation
 
-> <a class="install-extension-btn" href="vscode:extension/GitHub.copilot-chat">Install the GitHub Copilot Chat extension</a>
+GitHub Copilot Chat is a chat interface that lets you interact with GitHub Copilot, to ask and receive answers to coding-related questions. The chat interface provides access to coding information and support without requiring you to navigate documentation or search online forums.
 
-## Get your first Copilot chat conversation
+In this tutorial, you'll be creating a simple Node.js web application. So, let's get started and ask Copilot to tell us more about web frameworks for Node.js.
 
-As you're iterating over your code, there might be moments where you need help from your AI pair programmer to accomplish a task. In the [Copilot Getting Started](/docs/copilot/getting-started.md), you used Copilot inline suggestions and optionally provided context by using code comments.
+1. Open the Chat view from the Activity Bar or press `kb(workbench.action.chat.open)`.
 
-In some cases, it might be more efficient to just ask Copilot about a specific topic by using a chat message. For example, a common cause for errors is when a method receives invalid input. In this step, you start with a simple calculator in TypeScript, and use GitHub Copilot Chat to refactor the code to add error handling.
+    ![Screenshot of VS Code editor, showing the Copilot Chat view, highlighting the chat control in the Activity bar.](./images/getting-started-chat/copilot-chat-view.png)
 
-![Video that shows asking Copilot to add error handling to our code. Copilot presents a diff view with the suggested code changes.](./images/getting-started-chat/copilot-chat-add-error-handling.gif)
+    > **Tip**: Drag the Chat view to the Secondary Side bar to keep both the Chat view and Explorer view open.
 
-Let's use GitHub Copilot to help us add error handling to our code.
+1. In the chat input field, enter *nodejs web server frameworks*, and then press `kb(inlineChat.accept)` to send the request to Copilot.
 
-1. Open Visual Studio Code and create a new TypeScript file `Calculator.ts`.
+    Copilot returns a list of web frameworks and with some additional information. So, without leaving VS Code, you can search for coding information while you stay in VS Code. Note that your results might be different from what is shown in the screenshots.
```

Comment:
Should we give a bit of context about why your results might be a bit different (i.e. AI is non-deterministic)?